src/parser/jobs/rdm/Gauge.js

```diff
@@ -231,6 +259,71 @@ export default class Gauge extends Module {
 			}
 		}
 
+		_handleFinisher(abilityId) {
+			if (abilityId !== ACTIONS.VERFLARE.id && abilityId !== ACTIONS.VERHOLY.id) {
+				//Not a finisher, bail bail!
+				return
+			}
+
+			const isFireReady = this.combatants.selected.hasStatus(STATUSES.VERFIRE_READY.id)
+			const isStoneReady = this.combatants.selected.hasStatus(STATUSES.VERSTONE_READY.id)
+			const white = this._whiteMana
+			const black = this._blackMana
+			const isAccelerationUp = this.cooldowns.getCooldownRemaining(ACTIONS.ACCELERATION.id) === 0
+			let useVerHoly = false
+			let useVerFlare = false
+			let doesntMatter = false
+			let useOnBadProc = false
+
+			if (isStoneReady &&
```

Comment:
Lint's not flagging it so not going to block, but this suite of `if`s could do with some TLC